src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { act } from 'react'
+
+import { RenderResult, render, screen, userEvent, waitFor } from '../../test-utils'
+import { Tooltip } from './Tooltip'
+
+const defaultProps = {
+  title: 'Tooltip text',
+  onOpenChange: jest.fn(),
+  getPopupContainer: () => document.getElementById('popup-container') || document.body,
+}
+
+const renderTooltipWithWrapper = (props = defaultProps) : RenderResult => {
+  return render(
+    <div id="popup-container">
+      <Tooltip {...props}>{'Tooltip trigger'}</Tooltip>
+    </div>
```

Comment:
this container is useful for debugging tests, because without it the actual tooltip message is not rendered by tools like testing playground (because no "body" element is rendered)